.vscode/settings.json

```diff
@@ -9,6 +9,10 @@
     "files.trimTrailingWhitespace": false
   },
   "typescript.tsdk": "node_modules/typescript/lib",
+  "jest.autoRun": {
+    "enabled": false,
+    "watch": false
+  },
```

Comment:
Don't commit this 🤔

Reply:
The poblem is that vscode is not providing a `.vscode/private.setting.json` thing. And you are commiting your own settings to the repo as well.

Reply:
If you think that adding this file to `.gitignore` and removin it is ok. I will make a commit to do that.